tests/Fixer/FunctionNotation/MethodArgumentSpaceFixerTest.php

```diff
@@ -223,6 +223,23 @@ function xyz(
     );
 ",
             ),
+            'test half-multiline function gets full-multiline' => array(
```

Comment:
Other test cases use comments for this, but I think this will yield better error messages.